pigallery2.json

```
@@ -0,0 +1,56 @@
+{
+    "PiGallery2": {
+        "containers": {
+            "pigallery2": {
+                "image": "bpatrik/pigallery2",
+                "launch_order": 1,
+                "ports": {
+                    "4001": {
+                        "description": "PiGallery2 WebUI port",
+                        "host_default": 4001,
+                        "label": "WebUI port",
+                        "protocol": "tcp",
+                        "ui": true
+                    }
+                },
+                "volumes": {
+                    "/config": {
+                        "description": "Choose a Share for configuration",
+                        "label": "Config Storage"
+                    },
+                    "/db": {
+                        "description": "Choose a Share to save app database",
+                        "label": "DB Storage"
+                    },
+                    "/tmp": {
+                        "description": "Choose a Share for temporary files",
+                        "label": "Temp Storage"
+                    },
+                    "/images": {
+                        "description": "Choose a Share for Media",
+                        "label": "Media Storage"
+                    }
+                },
+                "environment": {
+                    "PUID": {
+                        "description": "Enter a valid UID to run Ombi as. It must have full permissions to all Shares mapped in the previous step.",
```

probably want to change that for this Rockon
```suggestion
                        "description": "Enter a valid UID to run PiGallery2 as. It must have full permissions to all Shares mapped in the previous step.",
```